### DISCUSSION TITLE
Help with network install

### DISCUSSION THREAD

**Discussion by TempHauns** [SchweizerischeBundesbahnen/netzgrafik-editor-frontend#302] [General]
I Installed Backend and Frontent, following documentation. I changed "server_name" in "nginx-angular.conf" to an IP, to make it accessible from other machines in the network. When I access it via browser (tried chrome and firefox) I am stuck on the "You are getting logged in..." screen, not getting prompted for a login.
docker compose ps show postgres & keycloak are running

version 2.7.16 on debian 12

would greatly appreciate help with the install

![image](https://github.com/user-attachments/assets/7305f4fb-5ac6-4cde-980f-be58b4933c5b)

**Comment by TempHauns** [comment]
I installed the editor on ubuntu-desktop, following the same steps.
I can open the website on the machine itself via the browser (http://localhost:4200). It's working as intended.
Changing name to an IP in <nginx-angular.conf> lets me call the website from within my network, but again, I am stuck with the window above. Would be great if I could access the website from other devices!

**Comment by aiAdrian** [comment]
What is the current state of the reported problems? @All is there someone with deep experience in such issues?

**Reply by chenkins** [reply]
Any logs/screenshot from Web Debugger Tools in browser?

**Comment by BabuBahir** [comment]
I think they now run only the frontend as `npm run start:standalone` which disables backend
